### PR TITLE
EPMLABSBRN-970: [BE] Postman collection update and actualize readme info

### PR DESCRIPTION
**Pull request by koylubaevnt**
* Updated README.md
* Updated swagger annotation in code
* Updated api.raml
* Fix error in PunctuationValidator.kt: add ё as allowed symbol

[EPMLABSBRN-XX](https://jira.epam.com/jira/browse/EPMLABSBRN-XX)

**Description**:

### CODE REVIEW

File: src/main/kotlin/com/epam/brn/controller/AdminController.kt [hunk 1]
```diff
@@ -81,7 +81,7 @@ class AdminController(
         .body(BaseResponseDto(data = studyHistoryService.getHistories(userId, from, to)))
 
     @GetMapping("/monthHistories")
-    @ApiOperation("Get month user's study histories")
+    @ApiOperation("Get month user's study histories by month and yea")
```
Comment (ElenaSpb) [inline src/main/kotlin/com/epam/brn/controller/AdminController.kt:84]: yea -> year

File: src/main/kotlin/com/epam/brn/controller/SeriesController.kt [hunk 2]
```diff
@@ -16,22 +17,25 @@ import org.springframework.web.bind.annotation.RestController
...
 
     @GetMapping
+    @ApiOperation("Get all series for group")
     fun getSeriesForGroup(@RequestParam(value = "groupId") groupId: Long): ResponseEntity<BaseResponseDto> {
         val listDto = seriesService.findSeriesForGroup(groupId)
         return ok(BaseResponseDto(data = listDto))
     }
 
     @GetMapping("{seriesId}")
+    @ApiOperation("Get serie for id")
```
Comment (ElenaSpb) [inline src/main/kotlin/com/epam/brn/controller/SeriesController.kt:31]: serie -> series

Comment (koylubaevnt) [reply src/main/kotlin/com/epam/brn/controller/SeriesController.kt]: Fixed

File: src/main/kotlin/com/epam/brn/controller/AdminController.kt (hunk 1 above)
Comment (koylubaevnt) [reply src/main/kotlin/com/epam/brn/controller/AdminController.kt]: Fixed

**Review by ElenaSpb — APPROVED** [review]

**Pull request merged**